bms-short-title.js

```diff
@@ -0,0 +1,16 @@
+// ==UserScript==
+// @name         BMS Remove title 'Outside Open - '
+// @namespace    https://github.com/outsideopen/userscripts
+// @version      0.1
+// @description  Remove's 'Outside Open - ' from HTML Title
+// @author       Andy Rusch <arusch@outsideopen.com>
+// @match        https://bms.kaseya.com/*
+// @grant        none
+// ==/UserScript==
+
+(function() {
+    'use strict';
+var newtitle = document.title;
```

Comment:
Could you add the 4 spaces to align lines 13-15 to line 12?

Reply:
ok done